galaxy.yml

```diff
@@ -0,0 +1,17 @@
+---
+namespace: confluent
+name: platform
+version: 6.2.0
```

Comment:
this variable will need to be updated for the 7.0.0 release... wonder if theres a way to make it easier

Reply:
there is a bumpconfig file that we can update with this

Reply:
If this can be improved, I will happily merge a PR to fix this into this PR. 😉